test/DebugInfo/ASTSection_ObjC.swift

```diff
@@ -13,7 +13,6 @@
 // REQUIRES: executable_test
 // REQUIRES: objc_interop
 
-// CHECK: Loaded module ASTSection from
```

Comment:
I removed these messages because they're redundant. Two lines below, we print that we're importing the `ASTSection`, so it's not really adding any information here.